examples/cpp-debug-workspace/.theia/tasks.json

```diff
@@ -0,0 +1,12 @@
+{
+    "tasks": [
+        {
+            "label": "compile example",
+            "type": "shell",
+            "command": "cd build && g++ -o a ../a.cpp -g",
+            "options": {
+                "cwd": "${workspaceFolder}"
```

Comment:
The task fails if the build directory does not exists.

```json
{
    "tasks": [
        {
            "label": "Compile C++ Example (Debug)",
            "type": "shell",
            "command": "mkdir build && cd build && g++ -g3 -O0 ../a.cpp -o a",
            "options": {
                "cwd": "${workspaceFolder}"
            }
        }
    ]
}
```

Reply:
`mkdir -p build`, so it doesn't fail if the directory already exists :)

Reply:
Thanks guys! I'll make the update